fstat/static/js/app.js

```diff
@@ -0,0 +1,51 @@
+$('input[name="daterange"]').daterangepicker({
```

Comment:
I'm not an expert on frontend, but isn't it a good idea to put your entire JS into a private scope?